src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java

```diff
@@ -495,7 +496,10 @@ else if (!TokenUtil.areOnSameLine(startToken, brace)) {
      * @param braceLine line content
      */
     private void validateEol(DetailAST brace, String braceLine) {
-        if (CommonUtil.hasWhitespaceBefore(brace.getColumnNo(), braceLine)) {
+        final String line = Pattern.compile("/\\*.*?\\*/")
+                        .matcher(braceLine)
+                        .replaceAll(str -> " ".repeat(Math.max(0, str.end() - str.start())));
```

Comment:
The lambda target is a MatchResult, not a string, please name it as such (`result` or `group` or something short and descriptive).

Reply:
inspections caught this one too :)

https://circleci-tasks-prod.s3.us-east-1.amazonaws.com/storage/artifacts/2233fdd7-79cb-481c-a64c-0df373c0d898/42c1d432-b7d3-4b71-92e3-7a00f428bb9c/0/home/circleci/project/target/inspection-results/StandardVariableNames.xml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQVFQINEOCZV4RWNG%2F20221118%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20221118T145253Z&X-Amz-Expires=60&X-Amz-Security-Token=IQoJb3JpZ2luX2VjED8aCXVzLWVhc3QtMSJHMEUCIAYH0fzhkkRYfFekNiXSIltKOLEJ3pfTqGoXzzvhzTXxAiEAkjDjVvO8X1m%2Fg1Tv68R5rvfdRz8A%2BEKkwmr5zMhuAhMqqwIISBADGgwwNDU0NjY4MDY1NTYiDFZwIOALGMDS88WtsiqIAvk8U18o7gE6XXc84ysn4QBwC7aFO9ADtgrsh%2Fjr8cod2PCrAKT83ap3%2FH0ekRc8sXDhf6ZREHMsbMe9ontxCPSpr5%2FQMyOPpUqchnFKbqredtzFubTv9Q2SJmphqbrncp4JIki5mF5VDuKbsLheHiod5uwq0ykaeriHi7N5zBgP5ixkFOQMdtkH5kUnEW6qw7KoWdC83O9yi9jlpEKn0J9qXseVOvjLvYcxpsjtcnp%2BKMKlb%2B7dy%2FqmeqU2DTI5reNw6%2FYgMW3uQqazTIrxcmxQLb%2Bft7JfWSk8XD7o5AK2KeLfoFshsrpDv7zCIRm7SwAErIVnLnYoprXyUPIUCASrXhp6lD0VtDDns96bBjqdAdzw8HqcXco%2BDBS%2FF7cmtAG%2Fl8bLB9cUVvZDYmJO%2FaAikAYNaM82cS8schLnIEtV%2BMkwMOcCGpHLCYzuCeb%2FtoCfpM%2BsKPvrwMlso93VaXWfGF7jvQcE9WEUdVNTQSj%2F%2FFVuDo9VxAYSL6TazRSdvestC1xpGTLetwJOQ8P9lEjV49b2w%2FM2LQESxjO3k5%2BUVog9IkllfRkQpRAFp5U%3D&X-Amz-SignedHeaders=host&x-id=GetObject&X-Amz-Signature=65c63d9110269d09811224d3e3da026312d9b437f7a50d71f8dcaca097d3f090

Reply:
Also, we have pitest failures around lambda function, we may not even need `Math.max`, please investigate.